chainerx_cc/chainerx/cuda/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 install(FILES
     cublas.h
+    cusolver.h
```

Comment:
Not a major issue but these are actually sorted alphabetically so it'd be nice if you could fix the placement of this header. Same goes for e.g. `cusolver.cc` below.

Reply:
Fixed.